### PR TITLE
docs: Edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/angular-compiler-options.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The pre-Ivy message extraction tooling generated a variety of legacy formats for
 These message formats have a number of issues, such as whitespace handling and reliance upon information inside the original HTML of a template.
 
 The new message format is more resilient to whitespace changes, is the same across all translation file formats, and can be generated directly from calls to `$localize`.
-This allows `$localize` messages in application code to use the same id as identical `i18n` messages in component templates.
+This allows `$localize` messages in application code to use the same ID as identical `i18n` messages in component templates.
```
Comment (AndrewKushnir) [inline aio/content/guide/angular-compiler-options.md:100]: I think changing "id" -> "ID" makes the content inconsistent with previous references above, see:

- legacy ids
- message ids

Can we revert this?

**Review by AndrewKushnir — APPROVED** [review]
Looks good, just left 1 comment regarding "id" vs "ID".

**Review by aikithoughts — APPROVED** [review]
LGTM, but consider addressing AndrewKushnir's comment.

Reviewed-for: global-docs-approvers

File: aio/content/guide/angular-compiler-options.md (hunk 1 above)
Comment (TeriGlover) [reply aio/content/guide/angular-compiler-options.md:100]: Fixed for consistency.

**Pull request merged**